docs/source/open_paths/open_path.rst

```diff
@@ -0,0 +1,236 @@
+
+Either option gives us the geometries of atom 1 for the first and last bead. These we will need to process in the following section to get the momentum distributions.
+
+Next in the xml file follows some generic information on the total steps and connection to a driver.
+
+.. code-block:: xml
+
+  <total_steps>100000</total_steps>
+    <prng>
+      <seed>3348</seed>
+    </prng>
+    <ffsocket mode='unix' name='driver'>
+      <address>mbx</address>
+    </ffsocket>
+
+Here we tell i-pi to use 64 beads and read a restart file, which corresponds to an equilibrated ring polymer structure, to initialize the simulation. The forces are just the ones returned by the driver.
+
+.. code-block:: xml
+
+  <system>
+    <initialize nbeads='64'>
+      <file mode='chk'> RESTART-64 </file>
+    </initialize>
+    <forces>
+      <force forcefield='driver'/>
+    </forces>
+
+This code block specifies the open path. In this case it is on atom 1, which is a hydrogen atom.
+
+.. code-block:: xml
+
+  <normal_modes>
+    <open_paths> [1] </open_paths>
+  </normal_modes>
+
+Lastly follows all information on the NVT ensemble. 
+
+.. code-block:: xml
+
+    <ensemble>
+      <temperature units='kelvin'>60.0</temperature>
+    </ensemble>
+    <motion mode='dynamics'>
+      <dynamics mode='nvt' splitting='baoab'>
+        <thermostat mode='pile_l'>
+          <tau units='femtosecond'> 100 </tau>
+        </thermostat>
+        <timestep units='femtosecond'>0.25</timestep>
+      </dynamics>
+    </motion>
+    </system>
+    </simulation>
+
+The full input file can be downloaded :download:`here <../_static/open_paths_config.xml>`. The xyz files for the :download:`first <../_static/open_paths.pos_00.xyz>` and `last <../_static/open_paths.pos_63.xyz>` beads are also provided. 
+
+********************************
+Processing the simulation output
+********************************
+
+The output geometries from the simulation are used to first calculate the distance between the first and last bead (in the code referenced as ``dist_H``). Then the radial estimator in equation :eq:`estimator` has to be constructed. Below is some code showing how this can be done.
+
+.. code-block:: python
+
+  import numpy as np
+
+  def radial_estimator(delta, qP_q1, sigP):
+      """
+      Calculate the radial estimator for the end-to-end distance
+
+      Args:
+      delta (one dimensional array of floats): distances for which to 
+      evaluate the estimator
+      qP_q1 (float or one dimensional array): distances between the
+      last and the first bead computed from the sampled snapshots
+      sigP (float): standard deviation of the Gaussians, 
+      sigP = sqrt(hbar^2 beta / (P m))
+
+      Returns:
+      N (one dimensional array): radial estimator as a 
+      function of delta (same length), is averaged over all qP_q1
+      distances
+      """
+      D, q = np.meshgrid(np.asarray(delta), np.asarray(qP_q1))
```

Comment:
Implement the analytical D -> 0 limit. Optionally, consider the small qP_q1 limit as well